flight/Modules/Logging/logging.c

```diff
@@ -169,6 +172,7 @@ static void loggingTask(void *parameters)
 
 	// Connect callbacks for UAVOs being logged on change
 	FlightStatusConnectCallbackCtx(UAVObjCbSetFlag, &flightstatus_updated);
+	SystemIdentConnectCallbackCtx(UAVObjCbSetFlag, &systemident_updated);
```

Comment:
I think you need to use an `if SystemIdentHandle()` (like for waypoint) otherwise it will crash if the UAVO is not present.